cargo-near-build/src/types/near/docker_build/subprocess/nep330_build_info.rs

```diff
@@ -93,6 +93,7 @@ impl From<BuildInfoMixed>
                 build_environment: value.build_environment,
                 source_code_snapshot: value.source_code_snapshot.as_url().to_string(),
                 contract_path: value.contract_path,
+                output_wasm_path: None,
```

Comment:
Shouldn't it be populated here?

Reply:
No. Output_wasm_path is output result that build tool populates into metadata, and is used only during verification, not during initial build specifically by cargo-near. (And this may be needed to support verification of contracts built in different languages). 
It was chosen over imperative variant with trying to enforce this value externally from docker container, because imperative variant requires more preconditions (near-sdk version, image version), and it makes less sense to try to enforce something that may not happen. 
Populating it only if it's fully supported as a build output value instead of build input looks simpler.